src/styles/stars.scss

```diff
@@ -0,0 +1,48 @@
+/* #region main container */
+.stars {
+  display: flex;
+  text-decoration: none;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+/* #endregion */
+
+/* #region stars */
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+/* #endregion */
+
+/* #region active star */
+.stars .stars__star:nth-child(-n + 4) {
```

Comment:
In this task you need describe variant which will be work for 1,2,3 stars when we add that modifier. 
Check SCSS loop and make more universal solution.